microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -2260,6 +2286,51 @@ Access and manipulate the profile:
 ---
 
 
+<!-- ------------------------------ -->
```

Comment:
Omit or include the "service workers" enum value (section Clearing service worker browsing data) for use by ClearBrowsingData?  Delete this new section?  Move the 1 link into an above tab-set?
Rendered section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-2733&tabs=dotnetcsharp#clearing-service-worker-browsing-data

Outline is at top of main PR page.

@novac42 @peiche-jessica @aluhrs13

Reply:
This is an addition to the existing CoreWebView2BrowsingDataKinds. I would not explicitly call this enum value out unless we do for the rest of the enum values. No strong opinion if we want to keep this as is.

Reply:
The existing article doesn't mention "datakinds", so this section, which focuses on 1 member of that enum, is the only place that this enum is represented, and this new spot might overemphasize this one new member.  

Maybe the enum overall should be better represented in the article, w/ less focus on this particular "serviceworkers" member.

Reply:
The result is good, b/c it creates a mini-section "Clearing browsing data" within the existing section/tab:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-2733&tabs=dotnetcsharp#user-data
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/a7c64a27-50e4-47f7-b8c8-d2a39d0dd1e8)